docs/topics/db/aggregation.txt

```diff
@@ -403,7 +403,9 @@ ratings exceeding 3.0.
 
 It's difficult to intuit how the ORM will translate complex querysets into SQL
 queries so when in doubt, inspect the SQL with ``str(queryset.query)`` and
```

Comment:
I would link "inspect the SQL" to the new documentation so that things don't have to be repeated here.